java_src/foam/dao/EnabledAwareDAO.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.dao;
+import foam.dao.*;
+import foam.mlang.MLang.*;
+import foam.mlang.predicate.Predicate;
+import foam.mlang.order.Comparator;
+import foam.dao.Sink;
+import foam.nanos.auth.EnabledAware;
+
+public class EnabledAwareDAO
+  extends ProxyDAO
+  implements EnabledAware
+{
+  public Sink select(Sink s, Integer skip, Integer limit, Comparator order, Predicate predicate) {
+    return super.select(s, skip, limit, order, EQ(this.EnabledAware.ENABLED, true));
```

Comment:
this could be a constant so that you don't need to create a new one for each call